packages/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -363,17 +379,9 @@ class BaseDescopeWc extends HTMLElement {
     );
   }
 
-  logger = {
-    error: (message: string, description = '') => {
-      // eslint-disable-next-line no-console
-      console.error(message, description, new Error());
-      this.#updateDebuggerMessages(message, description);
-    },
-    info: (message: string, description = '') => {
-      // eslint-disable-next-line no-console
-      console.log(message, description);
-    },
-  };
+  logger() {
```

Comment:
Why do we need to make it a function?
We can override it also when it's a property, no?

Reply:
it is mainly to be able to call base function for the debugger and console